CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+cmake_minimum_required (VERSION 3.2)
```

Comment:
Can this just be 3 or do we need something from 3.2 specifically?

Reply:
3.0 is fine too.